packages/filefeeds/src/filefeeds.ts

```diff
@@ -308,6 +327,14 @@ export class OneSchemaFileFeedsClass extends EventEmitter {
 
       case "init-succeeded": {
         this._iframeInitSucceeded = true
+        const sessionToken = eventData.sessionToken
+        if (this.#resumeTokenKey && sessionToken) {
```

Comment:
I think we should make both of these checks hard invariants, which would require `launch()` to always set `this.#resumeTokenKey` with the derived user-jwt.